### ISSUE TITLE
CocoaPods support

### ISSUE THREAD

**Issue by clindsay3** [ChimeHQ/UITestingPlus#3]
I still prefer CocoaPods over SPM, would love if this package were published there as well!

**Comment by mattmassicotte** [comment]
I really hear you on this. It was very painful migrating from Carthage to SPM, and I still miss some features sometimes.

But, the thing is, at this point, I feel like the ship has sailed. Many big projects are dropping support for other dependency management systems if they had it, and Apple themselves (of course), publish many packages as SPM-only.

All that said, if you want to put together and maintain a podspec,, I will publish it!

**Comment by clindsay3** [comment]
Yeah, I get it. But then I am forced to nuke my derived data folder one more time, and waste one more hour dealing with obscure errors, and I am reinvigorated to go down with the ship. 😅

I'll put together a podspec. 👍

**Comment by mattmassicotte** [comment]
Go for it! While I (nearly) never have derived data issues anymore, I do sometimes run into cases where I need to clear the package cache. And, of course, Xcode's handling of package loading/uploading does slow things down on project open (but I have 66 packages in one project...).

**Comment by clindsay3** [comment]
For example, I just had a successful build, then reset the package cache and rebuilt, and got all these errors, even though the packages look like they're there in the project navigator. 😔 Seems so flaky. Maybe there's just some weird configuration thing I need to fix and it'll all get better. 🤷‍♂️

<img width="516" alt="Screenshot 2023-01-28 at 8 03 51 AM" src="https://user-images.githubusercontent.com/227043/215223592-09f2e3e2-7ecd-4cff-a225-820e8a495a29.png">

**Comment by clindsay3** [comment]
FWIW, resetting package again and then building fixed the issue. But it seems like it's always something!

**Comment by mattmassicotte** [comment]
Yeah it really does. I **think** in this case, one single package is failing to build, and Xcode does a horrible job of communicating that via this particular view. I've gotten really good at diagnosing and dealing with SPM issues, just like in the past I was good at doing that with Carthage.

I just want to be really clear: I'm not saying it is better **at all**. I've just accepted our new overlords 😬

**Comment by clindsay3** [comment]
Haha yeah, there's definitely a certain amount of just learning the weird quirks of our new master. 😆
